### DISCUSSION TITLE
[Feature] Support for more output format CSV/HTML #270

### DISCUSSION THREAD

**Discussion by KingOfBugbounty** [projectdiscovery/httpx#1852] [Ideas] [answered]
1. We decided to create a future feature for generating HTML within httpx that will solve some people's problems. My friend [@gustavorobertux](https://github.com/gustavorobertux) and I created this tool in Go that is within this Git, with the Dracula theme in HTML.
2. [Feature] Support for more output formats: CSV/HTML #270.
3. GitHub Discussion: https://github.com/projectdiscovery/httpx/issues/270
4. Join our Discord server at https://discord.gg/projectdiscovery to discuss the idea on the #httpx channel.

### Please describe your feature request:
(https://github.com/KingOfBugbounty/KingOfBugBountyTips/blob/master/resolvhtml.go)

![image](https://github.com/user-attachments/assets/d210d0d8-f885-46de-9e5a-9b5c480aeb10)

![image](https://github.com/user-attachments/assets/24a5b215-be8c-46a8-980d-f2e1d26ae286)

### Describe the use case of this feature:
A clear and concise description of the feature request's motivation and the use-cases in which it could be useful.

**Comment by ehsandeep** [comment]
@KingOfBugbounty CSV format is already supported, about HTML - we just added UI dashboard support in latest release, see https://github.com/projectdiscovery/httpx/releases/tag/v1.6.7

```
 $ httpx -h csv

Usage:
  httpx [flags]

Flags:
   -csv  store output in csv format
```

**Reply by KingOfBugbounty** [reply, marked as answer]
bro, but it's not csv, it's html. Don't you think we can help by putting this function inside httpx? It helps a lot of hunters

**Reply by gustavorobertux** [reply]
> @KingOfBugbounty CSV format is already supported, about HTML - we just added UI dashboard support in latest release, see https://github.com/projectdiscovery/httpx/releases/tag/v1.6.7
> 
> ```
>  $ httpx -h csv
> 
> Usage:
>   httpx [flags]
> 
> Flags:
>    -csv  store output in csv format
> ```

But the new version exports the HTML to the cloud, I can't see it on my own infrastructure, right?
